src/utils/change.link.ts

```diff
@@ -17,3 +19,10 @@ export const createSummaryListLink = (
 export const createChangeLinkConfig = (href: string, text: string, dataEventId: string) => {
   return createSummaryListLink('Change', href, text, dataEventId);
 };
+
+/**
+ * Return a change link with the transactionId and submissionId included
+ */
+export const createChangeLinkWithIds = (link: string, transactionId: string, submissionId: string) => {
+  return getUrlWithTransactionIdAndSubmissionId(link, transactionId, submissionId);
```

Comment:
Now you've removed that error handling, this function only delegates to another. I would just get rid of it and call `getUrlWithTransactionIdAndSubmissionId()` directly.

Reply:
Here, we're exporting a method from `src` to `views` in a manner consistent with how other methods have been exported (see other exports in `app.ts`). To keep that consistency intact, I believe it's best we leave it like this. Happy to discuss.

Reply:
Ah yes, I see. Makes sense.